src/PackageManager.UI/Views/MainWindow.xaml.cs

```diff
@@ -51,10 +51,11 @@ private Task<bool> OnBeforeChange()
             var context = processes.PrepareContextForProcessesKillBeforeChange();
             if (context.IsExecutable)
             {
+                var processNames = string.Join(" and ", context.ProcessNames);
+
                 bool result = navigator.Confirm(
                     "Plugin Manager",
-                    "Plugin Manager is going to write to files that are holded by other executables. " + Environment.NewLine +
-                    "Do you want to kill all instances of these applications?"
+                    $"Plugin Manager will be writing to files that are currently in use.\r\n\r\nDo you want to stop {processNames}?"

```

Comment:
```suggestion
                    $"Plugin Manager will be writing to files that are currently in use.\r\n\r\nDo you want to stop all instances of {processNames}?"

```

This is really just a detail and I have no strong feelings about it. At the end I accept both forms, with and without this suggestion.

Reply:
I have no option. I'm not good at user texts, as I tend to put there too much information.
Think it's @RussKie's call 😎

Reply:
I suppose if there's only a single instance of an app, than it is the first message. Else the second ;))

Reply:
No problem 😎